packages/nuxeo-retention-package/src/main/resources/install/templates/retention/nuxeo.defaults

```diff
@@ -0,0 +1,5 @@
+## DO NOT EDIT THIS FILE, USE nuxeo.conf  ##
+nuxeo.template.includes=
+retention.target=.
+
+nuxeo.retention.mode=standard
```

Comment:
Will we have more than 2 modes? If no, I would probably go for a "boolean".

Reply:
Not for now.
As far as I was able to understand, we can have "booleans" but at the end of the day they are strings, in fact. This way it will map directly what is on our requirements, but I would also be ok with `nuxeo.retention.compliant` with "true" or "false"

Reply:
I find it easier to understand/read if it's just "enable or not the compliance mode" :)
Let's add people to the conversation! WDYT @efge @RSalem07 ?

Reply:
from my point of view, as we have only the compliance mode enable or disabled (standard) we can use boolean, which also avoid string comparaison any any typo error

Reply:
I'd have chosen `nuxeo.retention.compliance.enabled=true/false`